frontend/src/service/SessionService.ts

```diff
@@ -0,0 +1,31 @@
+import { UserCredentials, InvalidCredentialsError } from "@/types/UserCredentials";
+import { Session } from "@/types/Session";
+import axios, { AxiosResponse } from "axios";
+
+const CONFIG = {
+    baseURL: "API AUTH SERVER URL" + "/login", //Set from .env
```

Comment:
Endpoint will be called `session` from my understanding of [this change](https://github.com/StackScrubs/stuq/blob/5d414bfc875476fefea5c5f74744f3f091b02688/backend/src/main/java/com/github/stackscrubs/stuq/backend/controller/SessionController.java).